apps/challenges/models.py

```diff
@@ -80,6 +84,13 @@ def is_active(self):
         return False
 
 
+@receiver(post_save, sender=Challenge)
+def publish_message_on_change(instance, created=False, **kwargs):
```

Comment:
@taranjeet sorry for the delay, can you please help me figure out why `update_fields` is coming `None`. For now, I am trying [this](https://stackoverflow.com/questions/36719566/identify-the-changed-fields-in-django-post-save-signal) solution. Please see if its the correct approach.